### ISSUE TITLE
[A11yAudit] Helper text not programmatically associated with form input [II] - HTML

### ISSUE THREAD

**Issue by ssabo1** [sparkdesignsystem/spark-design-system#3907]
**User Story**
As Spark we want to 

**Notes:**
Paciello Audit on Spark Design System
QLC-12 Input - Helper Text 
Helper text not programmatically associated with form input

ISSUE DESCRIPTION
When form controls lack clear and programmatically associated labels, they will not convey adequate information about their purpose and functionality to users of assistive technologies, who will therefore find it difficult or impossible to complete and submit forms.
Helper text on 32 - Helper Text and 33 - Helper Text (error) is not programmatically associated with its related form control and will likely be missed by assistive technology users.

Also the aria-invalid attribute (used to indicate that the value entered into an input field does not conform to the expected format) is incorrectly set to true upon page load. This would technically be correct if the form control was required, but even in those circumstances it would lead to a confusing announcement by assistive technologies when the field is first encountered. Invalid state should ideally only be conveyed after the user has entered some information and it was validated/found to be invalid.

RECOMMENDATIONS
Additional instructions, caveats, and helper text must be associated with a form control; this is in addition to the control having a label (its “accessible name”). The method for associating such descriptive text with the control is to use the aria-describedby attribute on the control, which is currently used to associate an error message with the control. The aria-describedby attribute must be set to the id(s) of the element(s) containing the descriptive text. If the text is from multiple sources use a space separated list of id values 'aria-describedby="IDvalue1 IDvalue2"'.
Also, when the form control is not in error, set the aria-invalid attribute to false.

**AC**
HTML (katie fixed React in earlier issue) 
- aria-describedby on the input should contain the ID of the helper text container (if the helper text exists)
- all stories with helper text will have aria-described

**Comment by ssabo1** [comment]
Confirmed: Added to team backlog.
